### PR TITLE
Prepare Camunda Modeler 5.7 Release

### PR DESCRIPTION
**Pull request by philippfromme**
* update modeling dependencies
* update @codemirror dependencies
* update CHANGELOG
* update release info

electron_Azkl2p2Cfh

Related to https://github.com/camunda/camunda-modeler/issues/3361

### CODE REVIEW

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,20 +44,20 @@ export function ReleaseInfo(props) {
...
-          <h4>New replace menu UI in the BPMN editor</h4>
-          Easily find the replacement element you are looking for with the search feature in the menu.
+          <h4>Extended support for using FEEL in Camunda 8 BPMN</h4>
+          FEEL expressions can be used for script task implementation and as error code.
         </li>
         <li>
-          <h4>Camunda 8.2 and 7.19 preview</h4>
-          Select the upcoming Camunda versions for preview of the new features. This includes support for link events in Camunda 8.
+          <h4>Improved replace menu in BPMN editor</h4>
+          With an element selected press <kbd>R</kbd> to open the replace menu. Unlink an element template simply by replacing the element.
```
Comment (marstamm) [inline client/src/plugins/version-info/ReleaseInfo.js:60]:
```suggestion
          With an element selected, press <kbd>R</kbd> to open the replace menu. Unlink an element template simply by replacing the element.
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -44,20 +44,20 @@ export function ReleaseInfo(props) {
...
-          <h4>Improved validation infrastructure</h4>
-          Editor now supports errors and warnings as problem types. Contribute your own validators with <a href="https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin">linter plugins</a>.
+          <h4>New color picker in the BPMN editor</h4>
+          Color one or more elements using the new color picker that is accessible through the context menu.
         </li>
         <li>
-          <h4>New replace menu UI in the BPMN editor</h4>
-          Easily find the replacement element you are looking for with the search feature in the menu.
+          <h4>Extended support for using FEEL in Camunda 8 BPMN</h4>
+          FEEL expressions can be used for script task implementation and as error code.
```
Comment (marstamm) [inline client/src/plugins/version-info/ReleaseInfo.js:56]:
```suggestion
          FEEL expressions can be used for script task implementations and as error codes.
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 3]
```diff
@@ -44,20 +44,20 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Model all elements for Camunda 8</h4>
-          The BPMN editor now supports drawing all BPMN symbols while it also indicates which elements are not supported in the engine.
+          <h4>New form fields and more in the Form editor</h4>
+          The form editor now supports text area, datetime and image form fields. Add conditions to form fields to control visibility.
```
Comment (marstamm) [inline client/src/plugins/version-info/ReleaseInfo.js:48]: Let's use the oxford comma in accordance with the [Camunda Writting Style guide](https://confluence.camunda.com/display/HAN/Camunda+Writing+Style+Guide#CamundaWritingStyleGuide-Punctuation) 
```suggestion
          The form editor now supports text area, datetime, and image form fields. Add conditions to form fields to control visibility.
```

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 1 above)
Comment (marstamm) [reply client/src/plugins/version-info/ReleaseInfo.js:60]: Also, TIL about the `kbd` tag. Very cool

**Review by marstamm — COMMENTED** [review]
Looks good to me overall, I added some small grammar/punctuation suggestions

**Comment by philippfromme** [comment]
> but the "replace with template" is also new in camunda modeler

Oh, wow. I missed that! Will include it!

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 1 above)
Comment (philippfromme) [reply client/src/plugins/version-info/ReleaseInfo.js:60]: <kbd>Haha, it's my favorite!</kbd>

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 3 above)
Comment (philippfromme) [reply client/src/plugins/version-info/ReleaseInfo.js:48]: TIL about the Oxford comma. 🐂

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 4]
```diff
@@ -44,20 +44,20 @@ export function ReleaseInfo(props) {
...
-          <h4>Improved validation infrastructure</h4>
-          Editor now supports errors and warnings as problem types. Contribute your own validators with <a href="https://github.com/camunda/camunda-modeler-custom-linter-rules-plugin">linter plugins</a>.
+          <h4>New color picker in the BPMN editor</h4>
+          Color one or more elements using the new color picker that is accessible through the context menu.
         </li>
         <li>
-          <h4>New replace menu UI in the BPMN editor</h4>
-          Easily find the replacement element you are looking for with the search feature in the menu.
+          <h4>Element templates accessible through replace menu</h4>
+          Apply an element template directly through the replace menu without having to change the element type first. Unlink an element template simply by replacing the element. With an element selected, press <kbd>R</kbd> to open the replace menu.
```
[the remark this answers is not in the document]
Comment (philippfromme) [reply client/src/plugins/version-info/ReleaseInfo.js]: Good point. I adjusted it.

**Pull request merged**